### PR TITLE
Fix android status bar bottom shadow

### PR DESCRIPTION
**Pull request by dotpinheiro**

### CODE REVIEW

**Comment by calintamas** [comment]
Hi @dotpinheiro! Thanks for the PR.

I am thinking that setting elevation 0 when `state.isVisible` is false should be enough. 
Also, please remove any prettier formatting, as it's not used in the codebase.

**Comment by dotpinheiro** [comment]
Hi @calintamas.
I've created a listener because there was an animation bug when elevation was set to 0
Oh, sure I'll remove it ! (:

File: src/components/base/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const BaseToast = (props) => {
   const baseStyle = [
     styles.base,
     styles.borderLeft,
-    { borderLeftColor: props.color || undefined }
+    { borderLeftColor: props.color || undefined },
+    {elevation: props.hideElevation ? 3 : 0},
```
Comment (calintamas) [inline src/components/base/index.js]: Shouldn't this be the other way around?

Comment (dotpinheiro) [reply src/components/base/index.js]: Yep, sorry
I've forgot to rename

I've already fixed

File: src/index.js [hunk 2]
```diff
@@ -187,6 +202,7 @@ class Toast extends Component {
     return (
       <SuccessToast
         onClose={this.hide}
+        hideElevation={this.state.showElevation}
```
Comment (calintamas) [inline src/index.js]: Then this should be `showElevation`

File: src/index.js [hunk 3]
```diff
@@ -200,6 +216,7 @@ class Toast extends Component {
     return (
       <ErrorToast
         onClose={this.hide}
+        hideElevation={this.state.showElevation}
```
Comment (calintamas) [inline src/index.js]: As well as this

File: src/index.js [hunk 4]
```diff
@@ -65,12 +65,27 @@ class Toast extends Component {
...
+      showElevation: false,
       text1: '',
       text2: ''
     }
   }
 
+  componentDidMount() {
+    this.state.animation.addListener(({value}) => {
+      if (value === 1 && this.state.isVisible === true) {
+        this.setState({showElevation: false});
```
Comment (calintamas) [inline src/index.js]: showElevation: true when visible

File: src/index.js [hunk 5]
```diff
@@ -65,12 +65,27 @@ class Toast extends Component {
...
+      }
+      if (value === 0 && this.state.isVisible === false) {
+        this.setState({showElevation: true});
```
Comment (calintamas) [inline src/index.js]: showElevation: false when hidden

**Pull request closed without merging**